### PR TITLE
chore(vmaas-go): promote vmaas-lib

### PR DESCRIPTION
**Pull request by yungbender**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Review by jdobes — APPROVED** [review]

**Comment by yungbender** [comment]
Pls merge this after merging #1034

**Pull request merged**
